### ISSUE TITLE
Offer human-readable view of diffs

### ISSUE THREAD

**Issue by acka47** [hbz/oerworldmap#895]
Currently, commits/diffs aren't understandable for normal users. We should add a simplified human-readable view of a commit/diff. 

We should start with the diffs for a specific resource. First steps would be:
- Don't show subject URIs – as all diffs concern the specific resource in question.
- Use labels from https://github.com/hbz/oerworldmap/blob/master/conf/labels.properties instead of property URIs.

**Comment by literarymachine** [comment]
A first version is available on beta, see e.g. https://beta.oerworldmap.org/log/urn:uuid:6d96ae75-341b-415b-9ee2-8a340ae1bb6c. Feedback is welcome!

**Comment by acka47** [comment]
The general approach looks promising. Actually, I have several problems to understand how this works for me. When I click on the oldest commit I also see changes from commits afterwards. IMO this should be the other way round. When I click on the latest commit I see now changes at all. Also:

- Useful would be a feature to compare two different historical versions of the resource (as one has in wiki systems).
 - I think it is important to be able to link to a diff between two commits (whether they occured consecutively in the log history or with other commits inbetween). Currently the URL doesn't change at all when I click on a specific commit and when I copy the actual URL that is behind a link (e.g. https://beta.oerworldmap.org/resource/urn:uuid:6d96ae75-341b-415b-9ee2-8a340ae1bb6c?version=27358a2a0953fe297b5982367d86ecaaead11fbf) I don't get where I want to get.

**Comment by karindr** [comment]
Much better than now.

- There should be labels an each version showing its date and maybe author also in the middle and the right column.
- I would prefer strikethrough text with a red line crossing out the deleted words. This is more intuitive.

**Comment by literarymachine** [comment]
> Actually, I have several problems to understand how this works for me. When I click on the oldest commit I also see changes from commits afterwards.

The first version always compared the selected commit to the most recent (`HEAD`) version. Thus, clicking on the oldest commit showed all changes and clicking on the newest showed none.

> Useful would be a feature to compare two different historical versions of the resource (as one has in wiki systems)

I deployed a first draft of this to beta, see e.g. https://beta.oerworldmap.org/log/urn:uuid:6d96ae75-341b-415b-9ee2-8a340ae1bb6c. Use the radio buttons to select to versions to compare. The UI is obviously preliminary, @j0hj0h @formimkontext and I just wanted to check if the general approach appeals to you.

> I think it is important to be able to link to a diff between two commits (whether they occured consecutively in the log history or with other commits inbetween).

Observe the URL when selecting versions to compare, it now reflects them. Clicking on an actual version now takes you to that version in the system.

**Comment by acka47** [comment]
+1 This already is very helpful and works really well! We should deploy this for now and can then improve it. One thing to do is show the name of the editor with an underlying link to their profile instead of the email address. This can be done in a later step, though...

**Comment by karindr** [comment]
+1. Much better. I Would prefer crossed out instead of marked read, but with a little bit of thinking this is understandable too.

**Comment by trugwaldsaenger** [comment]
Good work! This seems to have been implemented on production allready. Can we close the issue?
Here come some minor ideas for the next iteration:
- refine overall layout of the page, maybe center the text instead of having it left-justified.
- add a "close page" button - at the moment it is necessary to close the new tab
- I added a logo - this was not marked green, though I guess it should have been.
But these are minor issues, so I guess we can do this later.
